framework/core/Project/Lang.pm

```diff
@@ -56,15 +57,34 @@ sub new {
     return $class->SUPER::new($PID, $name, $vcs);
 }
 
+sub printstack {
+    my ($package, $filename, $line, $subroutine, $hasargs, $wantarray, $evaltext, $is_require, $hints, $bitmask, $hinthash);
+    my $i = 1;
+    my @r;
+    while (@r = caller($i)) {
+        ($package, $filename, $line, $subroutine, $hasargs, $wantarray, $evaltext, $is_require, $hints, $bitmask, $hinthash) = @r;
+        print "$filename:$line $subroutine\n";
+        $i++;
+    }
+}
+
```

Comment:
Remove or move to Utils?

Reply:
moved to utils as part of Math pull request

Reply:
I merged the Math PR, so you can remove this function.